app/models/persistent_volume.rb

```diff
@@ -1,4 +1,6 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  include NewWithTypeStiMixin
+  has_one :persistent_volume_claim, :dependent => :destroy
```

Comment:
```ruby
has_one :persistent_volume_claim_bound, :class_name => 'PersistentVolumeClaim', :dependent => :destroy
```

Change to this, i think it would make more sense.